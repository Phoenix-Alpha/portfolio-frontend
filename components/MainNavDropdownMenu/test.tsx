import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { MainNavDropdownMenu } from '.'

describe('<MainNavDropdownMenu />', () => {
  it('should renders correct <MainNavDropdownMenu />', () => {
    const wrapper = shallow(<MainNavDropdownMenu />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
