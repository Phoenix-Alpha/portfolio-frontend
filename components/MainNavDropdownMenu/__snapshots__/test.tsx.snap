// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainNavDropdownMenu /> should renders correct <MainNavDropdownMenu /> 1`] = `
<div
  className="absolute bg-transparent w-auto shadow-sm transition-all duration-400 ease-in-out z-[9999] top-full invisible translate-y-[20px]"
>
  <div
    className="h-[20px] bg-transparent"
  />
  <Container
    className="flex justify-center items-center !px-0 mx-auto md:min-w-[150px] text-sm font-semibold bg-white rounded-sm flex-no-wrap"
  >
    <ul
      className="flex flex-col w-full"
    />
  </Container>
</div>
`;
